pytest_fixtures/core/sat_cap_factory.py

```diff
@@ -95,8 +169,10 @@ def module_capsule_configured(module_capsule_host, module_target_sat):
 @pytest.fixture(scope='session')
 def session_capsule_configured(session_capsule_host, session_target_sat):
     """Configure the capsule instance with the satellite from settings.server.hostname"""
-    session_capsule_host.capsule_setup(sat_host=session_target_sat)
-    yield session_capsule_host
+    if session_capsule_host:
+        session_capsule_host.capsule_setup(sat_host=session_target_sat)
+        yield session_capsule_host
+    yield
```

Comment:
commit this suggestion directly, instead of separate PR :)
```suggestion
    if session_capsule_host:
        session_capsule_host.capsule_setup(sat_host=session_target_sat)
        yield session_capsule_host
```

Reply:
@jyejare Can you check this asap as puppet capsule tests are still failing because of this?

Reply:
Changed!